File: demoFireBase/app/src/main/java/com/example/demofirebase/Model/SinhVien.java
package com.example.demofirebase.Model;

public class SinhVien {
    String hoTen;
    int msSinhVien;

    public SinhVien(String hoTen, int msSinhVien) {
        this.hoTen = hoTen;
        this.msSinhVien = msSinhVien;
    }

    public SinhVien() {
    }

    public String getHoTen() {
        return hoTen;
    }

    public void setHoTen(String hoTen) {
        this.hoTen = hoTen;
    }

    public int getMsSinhVien() {
        return msSinhVien;
    }

    public void setMsSinhVien(int msSinhVien) {
        this.msSinhVien = msSinhVien;
    }
}

File: demoRecycelKT/app/src/main/java/com/example/demorecycelkt/SinhVien.kt
package com.example.demorecycelkt

class SinhVien{
    private var HoTen: String = ""
    private var NamSinh: Int = 0

    override fun toString(): String {
        return "SinhVien(HoTen='$HoTen', NamSinh=$NamSinh)"
    }

    constructor(HoTen: String, NamSinh: Int){
        this.HoTen = HoTen
        this.NamSinh = NamSinh
    }

    constructor()

    fun setHoTen(HoTen : String){
        this.HoTen = HoTen
    }

    fun getHoTen() : String {
        return this.HoTen
    }

    fun setNamSinh(NamSinh : Int){
        this.NamSinh = NamSinh
    }

    fun getNamSinh() : Int {
        return this.NamSinh
    }
}